src/models/blockchain/block/validity/block_program.rs

```diff
@@ -179,11 +179,15 @@ pub(crate) mod test {
     use itertools::Itertools;
     use tasm_lib::triton_vm::vm::PublicInput;
     use tracing_test::traced_test;
+    use triton_vm::prelude::Digest;
 
     use super::*;
-    use crate::job_queue::triton_vm::TritonVmJobQueue;
+    use crate::job_queue::triton_vm::{TritonVmJobPriority, TritonVmJobQueue};
     use crate::models::blockchain::block::validity::block_primitive_witness::test::deterministic_block_primitive_witness;
+    use crate::models::blockchain::block::{Block, BlockPrimitiveWitness, TritonVmProofJobOptions};
```

Comment:
granularity

Reply:
Thanks for noticing. Apparently I haven't set up the right formatting rules on my desktop at home. Will fix.